source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -85,27 +91,32 @@ the field on the collection. The following snippet creats an index on the
    db.shipwrecks({coordinates: "2d"})
 
 See the 
-:manual:`MongoDB server manual page on legacy coordinate pairs </geospatial-queries/index.html#legacy-coordinate-pairs>`
+:manual:`MongoDB server manual page on legacy coordinate pairs </geospatial-queries/#legacy-coordinate-pairs>`
 for more information.
 
 .. note::
 
    Spherical (``2dsphere``) and flat (``2d``) indexes support some, but
    not all, of the same query operators. For a full list of operators
    and their index compatibility, consult the
-   :manual:`manual entry for geospatial queries </geospatial-queries/index.html#geospatial-models>`.
+   :manual:`manual entry for geospatial queries </geospatial-queries/#geospatial-query-operators>`.
 
 Examples
 --------
 
-The following examples use the ``theaters`` collection of the
-``sample_mflix`` sample database available in MongoDB Atlas, with a
-``2dsphere`` index on the ``location.geo`` field.
+The following examples use the MongoDB Atlas sample dataset. If you want to run
+the examples with the sample dataset, you can learn how to set up a free-tier Atlas
+cluster with the sample dataset by following our 
+:doc:`quick start guide </quick-start>`.
+
+The examples use the ``theaters`` collection of the ``sample_mflix`` database
+from the sample dataset. The ``theaters`` collection has a ``2dsphere`` index
```

Comment:
Nitpick: could be more specific. For example, does "I can has cheezburger" mean you can possess it or that you can eat it?

```suggestion
from the sample dataset. The ``theaters`` collection contains a ``2dsphere`` index
```

Reply:
Thanks for the note. I agree "contains" here is the more appropriate term. I will try to keep the possible ambiguity of the term "has" in mind going forward.